cogs/ocr.py

```diff
@@ -1,18 +1,22 @@
 import easyocr
 import cv2
 import difflib
-from .config import IMAGE_PATH, IMAGE_RESIZED, IMAGE_GRAY
+from .config import IMAGE_THRESH, IMAGE_PATH, IMAGE_RESIZED, IMAGE_GRAY, IMAGE_DISPLAYED
 from .utils import get_best_match_score, get_best_match_username
 
 
-def run_easyocr() -> list:
-    """This reads the uploaded image using
-    easyocr and returns the easyocr generated
-    list of tuples. It uses some preprocessing
-    techniques that were tuned for mtgo images
-    to improve accuracy."""
+def preprocess_image() -> object:
```

Comment:
I liked the concept of separating the preprocessing I saw in the code. I have since applied this here.